redi-connect-front/src/types/RedProfile.ts

```diff
@@ -40,7 +40,6 @@ export type RedProfile = {
 
   createdAt: Date
   updatedAt: Date
-  gaveGdprConsentAt: Date
```

Comment:
We've delegated the responsibility of setting this property from the front-end to the back-end. But it's still part of the model. So let's keep this property.

Reply:
ok